convert2rhel/breadcrumbs.py

```diff
@@ -88,7 +88,8 @@ def finish_collection(self, success=False):
 
         self._set_ended()
         self._save_migration_results()
-        self._save_rhsm_facts()
+        if not "CONVERT2RHEL_DISABLE_TELEMETRY" in os.environ:
```

Comment:
@hosekadam I think the conditions here should be (and all other too)

```suggestion
        if "CONVERT2RHEL_DISABLE_TELEMETRY" not in os.environ:
```

Reply:
yes, indeed. All the conditions should be changed to use `not in` instead of `not <thing> in <other_thing>`